playwright.config.js

```diff
@@ -23,7 +23,7 @@ const config = {
   testDir: './test/e2e',
   timeout: 30 * 1000,
   workers: process.env.CI ? 1 : undefined, // No parallel tests on CI
-  forbidOnly: !!process.env.CI, // Fail on CI if test.only in source
+  forbidOnly: Boolean(process.env.CI), // Fail on CI if test.only in source
```

Comment:
Is this enforced? The other one is more concise and well known.

Reply:
Yes. I believe it is enforced via ESLint's [no-extra-boolean-cast](https://eslint.org/docs/latest/rules/no-extra-boolean-cast) rule which is enabled when using their recommended settings. I could be Prettier, too.

The key point is that this change was not caused by custom settings in our lint configuration. We can look into overriding this setting if possible (Prettier is proudly infamous for being "opinionated" and therefore non-configurable), but my preference is to favor ESLint's and Prettier's recommended settings and then selectively enable additional rules that are not enabled by ESLint.

Reply:
Fixed.